networkx/algorithms/bipartite/covering.py

```diff
@@ -0,0 +1,67 @@
+"""
+********
+Covering
+********
+"""
+
+#    Copyright (C) 2015 by
```

Comment:
Copyright notices should go as close as possible to the head of the file (i.e. above the module-level documentation string). From the [FSF][1]:

> Put all the copyright notices together, right near the top of each file.

[1]: http://www.gnu.org/licenses/gpl-howto.en.html